test/test_lib.py

```diff
@@ -145,6 +145,27 @@ def test_list_with_fqbn(run_command):
     assert data[0]["library"]["compatible_with"]["arduino:avr:uno"]
 
 
+def test_list_all_with_fqbn(run_command):
+    assert run_command("update")
+
+    # Install core
+    assert run_command("core install arduino:avr")
```

Comment:
Is this just pulling the latest `arduino:avr` from https://downloads.arduino.cc/packages/package_index.json? If so, I think the platform version should be pinned to avoid it breaking if the platform's bundled libraries are changed.

Reply:
Yep, forgot to pin it!